### ISSUE TITLE
Storage Dictionary

### ISSUE THREAD

**Issue by william-silversmith** [seung-lab/cloud-volume#78]
@jabae was thinking of using Storage to upload to his bucket, but the bucket was structured like gs://BUCKET without a dataset or layer. 

We should make this use case more convenient while maintaining neuroglancer related safety.

**Comment by william-silversmith** [comment]
Nacho and Jingpeng mentioned the idea of using a dict interface at various points. Why not make it work like this?

**Comment by william-silversmith** [comment]
The dict idea works well if you only care about binary files. If you want JSON, downloading works well, but uploading is weird (how do you specify JSON?).

**Comment by nkemnitz** [comment]
What exactly was the dictionary idea and where does it come into play with arbitrary storage paths?

**Comment by william-silversmith** [comment]
The dictionary idea was that you could do something like:

```python
stor = Storage("gs://bucket/stairway/to/heaven/")
h5img = stor['temporal.h5']
stor['info'] = x
```
The implementation details aren't related to arbitrary paths, but the ideas are linked in terms of use case as it makes Storage a more convenient way to access any arbitrary cloud files.

**Comment by william-silversmith** [comment]
Resolved in #230
